lightning/checkpoints/checkpoints.go

```diff
@@ -457,17 +453,17 @@ func (cpdb *MySQLCheckpointsDB) Initialize(ctx context.Context, dbInfo map[strin
 
 	s := common.SQLWithRetry{DB: cpdb.db, Logger: log.L()}
 	err := s.Transact(ctx, "insert checkpoints", func(c context.Context, tx *sql.Tx) error {
-		// If `node_id` is not the same but the `table_name` duplicates,
+		// If `hash` is not the same but the `table_name` duplicates,
```

Comment:
Is `hash` still always `0` now (in L477)?

Reply:
For now still true